homeassistant/components/pegel_online/__init__.py

```diff
@@ -46,6 +43,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: PegelOnlineConfigEntry)
     return True
 
 
-async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+async def async_unload_entry(
+    hass: HomeAssistant, entry: PegelOnlineConfigEntry
+) -> bool:
```

Comment:
sorry for this very small unrelated change, but I tough it is still somehow related and further not worthy to create an own PR just for this single line :see_no_evil: